docs/zh/transform-v2/llm.md

```diff
@@ -118,3 +174,83 @@ sink {
 }
 ```
 
+### Customize the LLM model
+
+```hocon
+env {
+  job.mode = "BATCH"
+}
+
+source {
+  FakeSource {
+    row.num = 5
+    schema = {
+      fields {
+        id = "int"
+        name = "string"
+      }
+    }
+    rows = [
+      {fields = [1, "Jia Fan"], kind = INSERT}
+      {fields = [2, "Hailin Wang"], kind = INSERT}
+      {fields = [3, "Tomas"], kind = INSERT}
+      {fields = [4, "Eric"], kind = INSERT}
+      {fields = [5, "Guangdong Liu"], kind = INSERT}
+    ]
+    result_table_name = "fake"
+  }
+}
+
+transform {
+  LLM {
+    source_table_name = "fake"
+    model_provider = CUSTOM
+    model = gpt-4o-mini
```

Comment:
When we set `model_provider` to `CUSTOM`, do we also need set `model`?

Reply:
At present, it is not considered, in fact, deleting the model, apikey, and so on is also possible

Reply:
So I think we should remove `model` in example.

Reply:
The original idea was to use the model as a placeholder for ease of use. If you change the model, you don't need to change the following custom-config.

Reply:
oh I get it